### DISCUSSION TITLE
GitHub Code Search is experiencing a disruption [ Search failed. Please try again later ]- 10/11/2024

### DISCUSSION THREAD

**Discussion by LiteBrite82** [community/community#141212] [Other Feature Feedback, Questions, & Ideas]
Hi Community!

Unfortunately we are experiencing service disruptions with multiple products, including Code Search. 

Follow our [GitHub Status Page](https://www.githubstatus.com/incidents/myvz2tsj2dh8) for the latest updates. 

Please comment in this discussion if you are experiencing any possibly related issues with Code Search

**Comment by 3052** [comment]
for SEO, this is an example of the problem:

> Search failed. Please try again later.

https://github.com/search?q=hello&type=code

**Reply by LiteBrite82** [reply]
@3052 
Great call out! I added it to the title as well

**Comment by defonotxspy** [comment]
note:
everything works now

**Comment by fr51** [comment]
I'm still experiencing a disruption with Code Search. Since this morning (even though this bug is older), Code Search leads me to secondary rate limit when I'm not logged in

Steps to reproduce:

1. go to [GitHub homepage](https://github.com)
2. not being logged in, search something using the search field
3. wait until you land on a page saying you exceeded a secondary rate limit

Can anyone help/fix, please? Thanks in advance

**Reply by defonotxspy** [reply]
i dont think you can use code search while logged out, what would be the point of fixing that "bug"?

**Reply by fr51** [reply]
Prior to yesterday, I was able to use Code Search while being logged out. In addition, I'd expect to get an error saying I'm not logged in instead of a secondary rate limit (what's this, by the way?)

**Reply by defonotxspy** [reply]
oh well, in this case, i dont really know xd
i barely use github!
